tests/MongoDB.Driver.Core.Tests/Core/Operations/BulkMixedWriteOperationTests.cs

```diff
@@ -214,14 +216,21 @@ public void WriteConcern_should_work()
                     Hint = new BsonDocument("_id", 1)
                 }
             };
-            var subject = new BulkMixedWriteOperation(_collectionNamespace, requests, _messageEncoderSettings);
+            var subject = new BulkMixedWriteOperation(_collectionNamespace, requests, _messageEncoderSettings)
+            {
+                WriteConcern = writeConcern
+            };
 
-            var exception = Record.Exception(() => ExecuteOperation(subject, async));
+            var exception = Record.Exception(() => ExecuteOperation(subject, async, useImplicitSession: true));
```

Comment:
Why must `useImplicitSession` be `true`?

Reply:
I figured it out... some of tests now use `w:0` and you can't use unacknowledged writes with explicit sessions any more.